clean_files.txt

```diff
@@ -46,6 +46,7 @@ themes/command_duration.theme.bash
 themes/easy
 themes/modern
 themes/purity
+themes/candy
```

Comment:
You can place this alphabetically in the themes list, ie below `themes/brunton`

Reply:
I'm actually thinking of sorting the entire list as currently it is not sorted, but I think this will be outside of the scope for this PR. should I create a sort PR first?

Reply:
Yeah I see what you mean.
As the nature of this PR is pretty light, I would be okay if you sorted the themes list now.
If you're thinking of sorting all sections, I would probably wait and make that a new PR.